app/models/spree/gateway/amazon.rb

```diff
@@ -70,22 +70,61 @@ def authorize(amount, amazon_checkout, gateway_options={})
       if amount < 0
         return ActiveMerchant::Billing::Response.new(true, "Success", {})
       end
+
       order_number, payment_number = extract_order_and_payment_number(gateway_options)
       order = Spree::Order.find_by!(number: order_number)
       payment = Spree::Payment.find_by!(number: payment_number)
       authorization_reference_id = operation_unique_id(payment)
 
       load_amazon_mws(order.amazon_order_reference_id)
 
-      response = @mws.authorize(authorization_reference_id, amount / 100.0, order.currency)
-      if response["ErrorResponse"]
-        return ActiveMerchant::Billing::Response.new(false, response["ErrorResponse"]["Error"]["Message"], response)
+      response = nil
```

Comment:
Actually, sorry, I was reading that wrong. I think that might have been a leftover of a previous iteration? I've updated to use `response = begin ...` which I think is even better and avoid the nil assignment.  Thanks.